compiler/AST/symbol.cpp

```diff
@@ -3515,6 +3568,9 @@ VarSymbol* new_BoolSymbol(bool b, IF1_bool_type size) {
   imm.const_kind = NUM_KIND_BOOL;
   imm.num_index = size;
   VarSymbol *s;
+  // doesn't use uniqueConstantsHash because new_BoolSymbol is only
+  // called to initialize dtBools[i]->defaultValue.
+  // gTrue and gFalse are set up directly in initPrimitiveTypes.
```

Comment:
I looked at the code around how immediate Symbols work when I was considering not tracking them. At first it confused me why new_BoolSymbol doesn't have the same general pattern as the others. Hence this comment. It is meant to be part of this PR, but it's just a comment...